.github/workflows/release.yml

```diff
@@ -148,10 +148,10 @@ jobs:
       - name: Release
         run: |
           cd gradle-plugin
-          ./gradlew bintrayUpload
-        env:
-          BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
-          BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
-
+          TAG_NAME=${GITHUB_REF#refs/tags/}
 
+          ./gradlew :configure:publishPluginToS3 --tag-name=TAG_NAME
```

Comment:
Should this be

```suggestion
          ./gradlew :configure:publishPluginToS3 --tag-name=$TAG_NAME
```

?

Reply:
Yes, it should be, thanks for catching that! 🙇